src/get-started/install.md

```diff
@@ -84,6 +85,22 @@ You may also run the extended test suite located in the `sandbox` directory:
 cargo test --manifest-path sandbox/Cargo.toml
 ```
 
+Since version [0.2.0][rel0.2.0] Exonum supports RocksDB as alternative data storage.
+To enable RocksDB support you need to pass additional parameter to cargo:
+```shell
+cargo test --manifest-path exonum/Cargo.toml --features rocksdb
+```
+and for extended test suite:
+```shell
+cargo test --manifest-path sandbox/Cargo.toml --features rocksdb
+```
+
+If you want to use Exonum framework with RocksDB support as dependency in you project 
+you should add into `Cargo.toml` line like this:
```

Comment:
"...RocksDB support as dependency in you project you should add into `Cargo.toml` line like this:" -->
"RocksDB support as a dependency in your project, you should add the following line into `Cargo.toml`:"